client/menus/vehicles/options.lua

```diff
@@ -2108,8 +2108,7 @@ lib.registerMenu({
             lib.hideMenu(true)
             return
         end
-
-        SetVehicleExtra(cache.vehicle, args, checked)
+        SetVehicleExtra(cache.vehicle, args.extra, checked and 0 or 1)
```

Comment:
```suggestion
        SetVehicleExtra(cache.vehicle, args.extra, not checked)
```
You can add the change easily using this

Reply:
That's wired i have been making some stuff using extras and in the docs it said that it isn't a boolean.

Reply:
![image](https://github.com/BerkieBb/bMenu/assets/82737367/9bd122c6-d497-4741-845f-dfecb00f6106)
https://docs.fivem.net/natives/?_0x7EE3A3C5E4A40CC9